### PR TITLE
New Serbian Cheat Sheet

### PR DESCRIPTION
**Pull request by Bole032**
Instant Answer Page: https://duck.co/ia/view/serbian_cheat_sheet

edit: Changed the IA link.
edit2: Deleted the aliases.

### CODE REVIEW

**Comment by daxtheduck** [comment]
@Bole032 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by ManrajGrover** [comment]
@Bole032 Hey! :smile: Your build is failing because it is not able to locate IA Page with the ID `serbian_cheat_sheet`. Try creating a new page with name `Serbian Cheat Sheet`. This will fix the issue.

**Comment by Bole032** [comment]
Here you go : https://duck.co/ia/view/serbian_cheat_sheet

**Comment by Bole032** [comment]
I also changed it in the PR, wasn't sure were should I send the new link :P

File: share/goodie/cheat_sheets/json/serbian.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    "id": "serbian_cheat_sheet",
+    "name": "Serbian",
+    "description": "Basic Serbian words and phrases",
+
+    "metadata": {
+        "sourceName": "Serbian Phrasebook",
+        "sourceUrl" : "http://wikitravel.org/en/Serbian_phrasebooke"
+    },
+
+    "aliases": [
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/serbian.json]: Please could the `aliases` array be removed? It's not needed if it's empty.

**Comment by tagawa** [comment]
Hi @Bole032. Thank you very much for submitting this. The fixed link to the IA page looks good. I added one comment and when that's done the Travis build should work too.

**Comment by Bole032** [comment]
Do you want me to remove the aliases, or can you do it?

**Comment by tagawa** [comment]
@Bole032 Could you do it please, as part of the same pull request? Just commit and pushing (on the same branch) should be enough.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=serbian cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/serbian_cheat_sheet_mobile.png?aVv97pMz8Y)

**Comment by Bole032** [comment]
Is this ok?

**Comment by tagawa** [comment]
@Bole032 Yes! The checks have passed and it's on the beta server for testing: https://beta.duckduckgo.com/?q=serbian+phrases&ia=cheatsheet
I'll try to get some other native speakers to review it and give feedback.

**Comment by digitalnaSRB** [comment]
Yes, this phrases is good. They could goes.
https://beta.duckduckgo.com/?q=serbian+phrases&ia=cheatsheet

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=serbian cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/serbian_cheat_sheet_mobile.png?dm8V5yCgKN)

**Comment by tagawa** [comment]
Thanks for the quick review @digitalnaSRB.

And thank you for making your first Instant Answer @Bole032! This looks great so I'm merging and it should go live in a few days.

If you'd like to contribute a bit more, you're welcome to add more phrases to this cheat sheet in a new pull request, or you could try to fill some of the gaps in our Instant Answers for programmers: https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage

**Comment by daxtheduck** [comment]
@Bole032 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by Bole032** [comment]
Yes, I am very happy about my IA going live, and yes I planned on adding
sections like Traveling and Going out for a dinner.

**Pull request merged**